### PR TITLE
LOOP-1869 Valid Sensor session changes

### PR DESCRIPTION
**Pull request by ps2**

### CODE REVIEW

File: G4ShareSpy/G4CGMManager.swift [hunk 1]
```diff
@@ -64,7 +64,16 @@ public class G4CGMManager: CGMManager, ReceiverDelegate {
     }
 
     private(set) var latestReading: GlucoseG4?
-
+    
+    public var hasValidSensorSession: Bool {
+        // No way to detect valid/invalid session here.
+        return true
+    }
```
Comment (nhamming) [inline G4ShareSpy/G4CGMManager.swift:71]: Again, it might be nice to report `unknown` instead of `yes`

**Review by nhamming — APPROVED** [review]
LGTM

**Review by darinkrauss — APPROVED** [review]
I agree with Nate's comment, but otherwise LGTM! 🍔

**Review by rickpasetto — APPROVED** [review]
LGTM

**Pull request merged**